src/test/codegen/mem-replace-direct-memcpy.rs

```diff
@@ -18,7 +18,7 @@ pub fn replace_byte(dst: &mut u8, src: u8) -> u8 {
 // CHECK-NOT: call void @llvm.memcpy
 // CHECK: ; core::mem::replace
 // CHECK-NOT: call void @llvm.memcpy
-// CHECK: call void @llvm.memcpy.{{.+}}({{i8\*|ptr}} align 1 %{{.*}}, {{i8\*|ptr}} align 1 %dest, i{{.*}} 1, i1 false)
```

Comment:
ping @eddyb since you added this test.

Are the call arguments here relevant for the test? Especially the `%dest` and `%src`?

Reply:
On the assumption that the test was overly specific I stubbed out out the argument names.